docs/extras/guide/expense_receipts_v5.md

```diff
@@ -193,6 +193,16 @@ The following fields are extracted for Receipt V5:
 ## Purchase Category
 **category** ([ClassificationField](#classificationfield)): The purchase category among predefined classes.
 
+> Possible values include:
+> - toll
+> - food
+> - parking
```

Comment:
What does the `>` character do?

Reply:
Pretty grey comment block :]

Reply:
Hmm not sure that's the best representation, since it's not a quote.
Just a normal list would be better, no? But not quite sure I understand the reasoning.

Reply:
It's just for the display on the documentation. Since we're not really likely to use quotes, I thought it'd make it pop-out more. I can remove it, no big deal

Reply:
Maybe make "Possible values" into a heading, or at least in bold?

Just that it's usually better to follow standard tag semantics, in particular for accessibility reasons.

Reply:
Alright :+1: